pom.xml

```
@@ -27,7 +31,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-validation</artifactId>
 		</dependency>
-
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>8.0.22</version>
```

To się często zmienia. Wynieś proszę do zmiennej (properties):
```suggestion
			<version>${mysql.version}</version>
```

---

Current solution is totally fine for now. There is another ticet to replace all hard-coded versions with values from `.properties`.